lectures/solow.md

```diff
@@ -209,7 +207,7 @@ plot45(kstar)
 From our graphical analysis, it appears that $(k_t)$ converges to $k^*$, regardless of initial capital
 $k_0$.
 
-This is a form of global stability.
+This is a form of [global stability](https://intro.quantecon.org/scalar_dynam.html#global-stability).
```

Comment:
See comment above

Reply:
I have revised these two internal links. Thanks for pointing out.